ocsp/src/main/java/io/nats/ocsp/OcspExample.java

```diff
@@ -74,39 +66,28 @@ public static SSLContext getSiloedContext()
         X509ExtendedKeyManager keyManager =
                 PemUtils.loadIdentityMaterial(ocspClientCertPath(), ocspClientKeyPath());
 
-        // The example ca cert and key is in a pem file. loadCertificates loads the certificates from
-        // that pem file. You likely will have to figure out how to load your own certificates
-        // Then add the certificates to a Key Store. You will likely have a different way to make or
-        // load your own Key Store
-        List<X509Certificate> certificates = loadCertificates(ocspCaCertPath());
-        KeyStore keyStore = KeyStore.getInstance(KEYSTORE_TYPE);
-        keyStore.load(null, KEY_STORE_PASSWORD.toCharArray());
-        for (X509Certificate certificate : certificates) {
-            String alias = certificate.getSubjectX500Principal().getName();
-            keyStore.setCertificateEntry(alias, certificate);
-        }
+        // The example ca cert and key is in a pem file. This PemUtils makes a Trust Manager from it.
+        // You likely will have to figure out how to make your own Trust Manager
+        X509ExtendedTrustManager trustManager =
+                PemUtils.loadTrustMaterial(ocspCaCertPath());
```

Comment:
Since I'm going to make a similar comment below. It would be nice to use your utils to load the keystore. This was the majority of the code I copied because it was private in the utils.